packages/mui-base/src/FormControlUnstyled/FormControlUnstyled.tsx

```diff
@@ -58,9 +58,7 @@ export type FormControlUnstyledOwnerState = Omit<
  *
  * - [FormControlUnstyled API](https://mui.com/api/form-control-unstyled/)
  */
-const FormControlUnstyled = React.forwardRef(function FormControlUnstyled<
-  D extends React.ElementType = FormControlUnstyledTypeMap['defaultComponent'],
->(props: FormControlUnstyledProps<D>, ref: React.ForwardedRef<any>) {
```

Comment:
the `OverridableComponent<FormControlUnstyledTypeMap>` already handle the types (except `component` prop).